### ISSUE TITLE
Support kvxopt

### ISSUE THREAD

**Issue by DManowitz** [qpsolvers/qpsolvers#170]
The [kvxopt](https://github.com/sanurielf/kvxopt) package is an alternative to cvxopt that is also available from conda-forge.  However, it conflicts with cvxopt, so both cannot be installed.  Having a version of this package available on conda-forge that supports kvxopt would be appreciated.

**Comment by stephane-caron** [comment]
Thanks for mentioning this new solver! Do you know why the two solvers can't co-exist?

In general `qpsolvers` connects to any available solver, so we can add a solver interface for kvxopt without jeopardizing the one for cvxopt.

> Having a version of this package available on conda-forge that supports kvxopt would be appreciated.

This point is more specific to the conda-forge distribution of qpsolvers, which is maintained by @Tobias-Fischer. Can you open an issue in the corresponding issue tracker? https://github.com/conda-forge/qpsolvers-feedstock/issues

**Comment by DManowitz** [comment]
You're right.  I downloaded the kvxopt package from conda-forge directly, and it installs to a kxvopt directory, so I don't know why the package is listed as incompatible with cvxopt.  I'll file an issue under the kvxopt conda-forge feedstock.
